pyriemann/datasets/simulated.py

```diff
@@ -78,8 +80,8 @@ def make_masks(n_masks, n_dim0, n_dim1_min, rs):
 
 
 def make_gaussian_blobs(n_matrices=100, n_dim=2, class_sep=1.0, class_disp=1.0,
-                        return_centers=False, random_state=None, *,
-                        mat_mean=.0, mat_std=1., n_jobs=1,
+                        return_centers=False, center_dataset=True,
+                        random_state=None, centers=None, *, n_jobs=1,
```

Comment:
```suggestion
                        return_centers=False, random_state=None, *,
                        centers=None, mat_mean=.0, mat_std=1., center_dataset=False, n_jobs=1,
```
I suggest a modification ensuring full backward compatibility for `make_gaussian_blobs`.
When `centers=None` (default), first center `C0` is not defined as identity, but it is computed as previously using `mat_mean` and `mat_std` parameters.

```
    if centers is None:
        C0 = generate_random_spd_matrix(
            n_dim,
            random_state=random_state,
            mat_mean=mat_mean,
            mat_std=mat_std
        )
        epsilon = np.exp(class_sep / np.sqrt(n_dim))
        C1 = epsilon * C0
```

Moreover, `center_dataset` should be False by default.

Reply:
I see what you mean, but the problem is that with your solution we won't have full control of the distance between `C0` and `C1`. Yeah, of course, the `epsilon` is a proxy... but it is not the *exact* value for it. In my current proposal, the `class_sep` is exactly the distance between the two class. This is only possible because we fix `C0` to the Identity and then modulate the distance of `C1` with a stretching. 

Also, the `mat_mean` and `mat_std` parameters are parameters which are totally fine for generating random matrices, but we don't know exactly what to expect from the distribution of the matrices that will get sampled with them. This is why I thought they could be dropped (and too bad for backward compatibility)